instagram/src/pages/PostModal.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import PostAfterSelect from "../components/Post/PostAfterSelect";
+import PostBeforeSelect from "../components/Post/PostBeforeSelect";
+
+function PostModal({ setShowModal }) {
+  const [imgSrc, setImgSrc] = useState("");
+  const [selected, setSelected] = useState(false);
+  const close = () => setShowModal(false);
```

Comment:
```suggestion
  const onClose = () => setShowModal(false);
```
이벤트 핸들러는 단순 close보다는 prefix로 on 또는 handler를 추가해주는게 좋아보임
보통 props로 넘겨야된다하면 on을 붙이고, 곧바로 사용한다하면 handler를 붙이는 편

Reply:
props: onClose
바로사용: closeHandler

Reply:
ㅋㅋㅋㅇㄴ handlerClose

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ aha